internal/schema1/schema1.go

```diff
@@ -214,9 +214,10 @@ type MappedVirtualDiskController struct {
 
 // GuestDefinedCapabilities is part of the GuestConnectionInfo returned by a GuestConnection call on a utility VM
 type GuestDefinedCapabilities struct {
-	NamespaceAddRequestSupported bool `json:",omitempty"`
-	SignalProcessSupported       bool `json:",omitempty"`
-	DumpStacksSupported          bool `json:",omitempty"`
+	NamespaceAddRequestSupported  bool `json:",omitempty"`
```

Comment:
Oh we need to move this to schema2 for the future. Capabilities wasnt a schema1 thing

Reply:
This will be made in a separate PR.